pkg/virt-launcher/virtwrap/device/sriov/hostdev_test.go

```diff
@@ -195,3 +329,35 @@ func (p *stubPCIAddressPool) Pop(_ string) (string, error) {
 
 	return address, nil
 }
+
+type deviceDetacherStub struct {
+	fail bool
+}
```

Comment:
We don't want to use mocks instead of stub right ? kubevirt has the mocks autogenerated tool implemented.